src/scales/interpolatedColorScale.ts

```diff
@@ -10,7 +10,7 @@ export module Scales {
    *
    * By default it generates a linear scale internally.
    */
-  export class InterpolatedColor extends Scale<number, string> {
+  export class InterpolatedColor extends Scale<number, number> {
```

Comment:
After diving into the code, it forced me to make this contract.  Looking at it again, this does look odd.  @rcchen do you have any context on why the compiler would force me to make this decision?